assets/javascripts/modules/analytics/cmp.js

```diff
@@ -34,4 +37,10 @@ const checkCCPA = () => new Promise((resolve) => {
 
 const registerCallbackOnConsentChange = (fn) => onConsentChange(fn);
 
-export { getConsentForVendors, checkAllTCFv2PurposesAreOptedIn, checkCCPA, registerCallbackOnConsentChange };
+const createPrivacySettingsLink = () => {
+    const privacySettingsButton = document.querySelector(PRIVACY_SETTINGS_SELECTOR);
+    privacySettingsButton.classList.remove(HIDDEN_CLASS);
```

Comment:
Can we wrap the `classList.remove` and `addEventListener` calls in a conditional `if (privacySettingsButton) { ... }`? Just in case someone removes the link from the template, or updates the class names...